hvpy/api_groups/jpeg2000/tests/test_get_jpx_closest_to_mid_point.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+
+import pytest
+from pydantic import ValidationError
+
+from hvpy.api_groups.jpeg2000.get_jpx_closest_to_mid_point import getJPXClosestToMidPointInputParameters
+from hvpy.core import execute_api_call
+
+startTimes = [datetime(2014, 1, 1, 0, 0, 0), datetime(2014, 1, 1, 2, 3, 3), datetime(2014, 1, 1, 4, 5, 5)]
+endTimes = [datetime(2014, 1, 1, 0, 45, 0), datetime(2014, 1, 1, 2, 33, 3), datetime(2014, 1, 1, 4, 54, 5)]
+
+
+@pytest.mark.parametrize(
```

Comment:
Same here, can you undo these parametrize?

Reply:
Sure. But can you explain what you said in that comment?

Reply:
We can worry about that later on.